api/src/paths/publish/survey.ts

```diff
@@ -0,0 +1,119 @@
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { SYSTEM_ROLE } from '../../constants/roles';
+import { getDBConnection } from '../../database/db';
+import { authorizeRequestHandler } from '../../request-handlers/security/authorization';
+import { PlatformService } from '../../services/platform-service';
+import { getLogger } from '../../utils/logger';
+
+const defaultLog = getLogger('/api/publish/survey');
+
+export const POST: Operation = [
+  authorizeRequestHandler(() => {
+    return {
+      and: [
+        {
+          validSystemRoles: [SYSTEM_ROLE.SYSTEM_ADMIN, SYSTEM_ROLE.PROJECT_CREATOR, SYSTEM_ROLE.DATA_ADMINISTRATOR],
+          discriminator: 'SystemRole'
+        }
+      ]
+    };
+  }),
```

Comment:
I think we'll need to alter this a bit. I think we probably want to allow project leads to submit, but not project creators. (A project creator gets added as the initial lead of a project, so they will typically have the permission anyways, indirectly).

`[system_admin, data_admin] OR [project_lead]`